topology.yaml

```diff
@@ -70,3 +70,7 @@ start args:
     --predictable:
         action: store_true
         help: If specified, attempt to expose container ports to the same port number on the host
+    --spark2-version:
+        default: 2_1-r1
+        help: If specified, the version of Spark2 to install from archive.cloudera.com
```

Comment:
Same as above. Since this is not just cloudera-specific, shouldn't this be handled differently?

Reply:
IGNORE ME AGAIN!